### DISCUSSION TITLE
参考文献设置

### DISCUSSION THREAD

**Discussion by ZJJ0106** [mengchaoheng/SCUT_thesis#48] [Q&A]
请问 参考文献的排列如何改成按照姓氏排名

**Comment by mengchaoheng** [comment]
不清楚，目前没这个需求

**Comment by ZJJ0106** [comment]
引用的时候如何更改成姓名（年份），上角标是脚注呢

**Reply by note286** [reply]
这是你的需求还是学校的需求还是导师的需求。

**Reply by ZJJ0106** [reply]
学校的啊

**Reply by note286** [reply]
来自研究生学位论文撰写规范二○二二年十一月修订

![image](https://github.com/mengchaoheng/SCUT_thesis/assets/16207292/ad1f192d-3206-4c9c-adb6-937474a50e30)

![image](https://github.com/mengchaoheng/SCUT_thesis/assets/16207292/669337a2-46f8-4c80-bf62-b8dfcd9a1bff)

**Comment by ZJJ0106** [comment]
学校的啊
